core/src/main/java/com/javadiscord/jdi/core/Discord.java

```diff
@@ -175,6 +175,8 @@ public void stop() {
             EXECUTOR.shutdownNow();
             Thread.currentThread().interrupt();
         }
+
+        discordRequestDispatcher.setAppIsActive(false);
```

Comment:
This needs to be called **before** shutting down the executor to make sense.
Also I wouldnt call make it a setter, because setting it active again does nothing. 
Instead maybe create a simple stop or shutdown method in that as well.